charts/posthog/values.yaml

```diff
@@ -16,7 +16,7 @@ image:
   # -- PostHog image tag to use (example: `release-1.43.0`).
   tag:
   # -- PostHog default image. Do not overwrite, use `image.sha` or `image.tag` instead.
-  default: ":release-1.43.0"
+  default: ":latest"
```

Comment:
we have stopped pushing `lastest`, let's use `master` here?

Reply:
this is posthog/posthog, where we push latest but not master by the looks of it https://hub.docker.com/r/posthog/posthog/tags?page=1&name=master